discojs/src/aggregator/base.ts

```diff
@@ -134,6 +154,16 @@ export abstract class Base<T> extends EventEmitter<{'aggregation': T }> {
     return false
   }
 
+  /**
+   * Remove a node's id from the set of active nodes.
+   * @param nodeId The node to be removed
+   */
+  removeNode (nodeId: client.NodeID): void{
+    if (this.nodes.has(nodeId)) {
+      this._nodes = this._nodes.delete(nodeId)
+    }
```

Comment:
`delete` doesn't fail if the value is not present (and it avoid double lookup)

```suggestion
    this._nodes = this._nodes.delete(nodeId)
```